packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -285,6 +285,10 @@ export const generateDefaultDeviceMenuProps = (
                 key: camera.id,
                 text: camera.name,
                 title: camera.name,
+                iconProps: {
+                  iconName: 'ContextMenuCameraIcon',
```

Comment:
From the PR description, what needed changed on the devices button? From the screenshot I only see a change to the text of the camera and microphone buttons

Reply:
Contextual menu order is changed, and added icons to mic/camera device selection menu